uat/features/stanford_date_formats.feature

```diff
@@ -1,10 +1,11 @@
+@stanford
 Feature: Stanford Date Formats
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford Date Formats module is working properly
 
 
-  @api
+  @api @safe @deploy
```

Comment:
Same comment here as for context_list_active, but we actually ought to enable stanford_date_formats in the install profile.